frontend/src/components/failures-list/index.spec.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import jasmineEnzyme from 'jasmine-enzyme'
+import { mount } from 'enzyme'
+import EventsList from 'components/events-list'
```

Comment:
This should be `FailureList`. 
And this test is not testing the day split rendering... I think that's the most important part to be tested in this component.